Makefile

```diff
@@ -348,23 +348,22 @@ contribs/gmf/build/angular-locale_%.js: package.json
 	$(PY_VENV_BIN)/pip install -r requirements.txt
 	touch $@
 
+.build/applications.timestamp: ci/applications.yaml ci/applications-versions.yaml
+	c2cciutils-download-applications --applications-file=ci/applications.yaml --versions-file=ci/applications-versions.yaml || \
+		(echo You should install c2cciutils >= 1.3 && false)
```

Comment:
```suggestion
		(echo 'You should install c2cciutils : pip3 install --user "c2cciutils>=1.3"'  && false)
```

Reply:
Note that if checks and/or dependencies vary between versions of c2cciutils, it will be hard to keep a version of c2cciutils on host that match with all projects.

Maybe we should use a venv per project.

But for me we should give information about how to install c2cciutils.

Reply:
Done :-)